### PR TITLE
feat: add environment variable override for client_max_body_size

### PR DESCRIPTION
**Pull request by marc-ste**
[CATINV-618](https://deliveroo.atlassian.net/jira/software/c/projects/CATINV/boards/2020?quickFilter=4703&selectedIssue=CATINV-618)

[CATINV-618]: https://deliveroo.atlassian.net/browse/CATINV-618?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ethicalaakash** [comment]
The change looks good to me but the build is failing. Can you please look into that.

File: start.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ fi
   -e "s/<CLIENT_BODY_BUFFER_SIZE>/${CLIENT_BODY_BUFFER_SIZE:-8k}/g" \
   -e "s:<PROXY_TIMEOUT>:${PROXY_TIMEOUT:-60s}:g" \
   -e "s:<NGINX_KEEPALIVE_TIMEOUT>:${NGINX_KEEPALIVE_TIMEOUT:-20s}:g" \
+  -e "s:<NGINX_CLIENT_MAX_BODY_SIZE>:${NGINX_CLIENT_MAX_BODY_SIZE:-5m}:g" \
```
Comment (ethicalaakash) [inline start.sh:22]: Shouldn't this be `CLIENT_MAX_BODY_SIZE`

**Review by ethicalaakash — APPROVED** [review]
LGTM

**Review by mathildathompson — APPROVED** [review]
LGTM 👍

**Pull request merged**